src/interfaces/coral_web/src/components/Shared/DragDropFileInput/DragDropFileInput.tsx

```diff
@@ -3,6 +3,9 @@ import { ReactNode, useState } from 'react';
 import { Text } from '@/components/Shared/Text';
 import { cn } from '@/utils';
 
+// Hidden files that should not be uploaded
+const IGNORED_FILES = ['.DS_Store'];
```

Comment:
Feel free to suggest more

Reply:
interesting that this is being selected from the file picker